### ISSUE TITLE
H shaped chin dimple

### ISSUE THREAD

**Issue by poratshay** [obophenotype/human-phenotype-ontology#11574]
**Preferred term label:**
H shaped chin dimple

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID 20012803

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:2053

**Your nano-attribution (ORCID)**
0000-0001-9947-035X

**Comment by pnrobinson** [comment]
@poratshay thanks!
This was Chin with H-shaped crease 
HP:0011824
- I added H shaped chin dimple as a synonym
